pythonforandroid/recipes/python3/__init__.py

```diff
@@ -55,6 +55,8 @@ class Python3Recipe(TargetPythonRecipe):
     name = 'python3'
 
     patches = [
+        ('patches/pyconfig_detection.patch'),
+
```

Comment:
Do we need to patch python3 recipe as well? I understand that the problem is with hostpython and MacOs during build stage...:thinking:

Reply:
Yes I also thought, but then I saw the build failing with the exact same issue so I decided to add it there as well